README.md

```diff
@@ -2,23 +2,30 @@
 
 > #### 🚧 Fleet GitOps is currently in development.
 >
-> In the mean time, you can explore this repo for a preview of Fleet's GitOps best practice, and track its progress [here](https://github.com/fleetdm/fleet/issues/13643).
+> In the meantime, you can explore this repo for a preview of Fleet's GitOps best practice, and track its progress [here](https://github.com/fleetdm/fleet/issues/13643).
 
 This folder illustrates the best practice for using Fleet with a GitOps workflow.
 
-How to setup a GitOps workflow to manage Fleet:
+How to set up a GitOps workflow to manage Fleet:
 
 1. Clone this repository.
 
 2. Make any changes to policies, queries, controls, and more defined in the files.
 
-3. Create your own GitHub repository and push your cloned and modified code to your repo.
+3. Create your own GitHub repository.
 
-4. Add Fleet's open-source [GitHub action](https://github.com/fleetdm/fleet-gitops) to your repo. Now, when anyone makes a change to any file, the action will run and update Fleet.
+4. Add `FLEET_URL` and `FLEET_API_KEY` to your GitHub repository secrets.
+
+5. Update the `env` section of your [.github/workflows/workflow.yml](https://github.com/fleetdm/fleet-gitops/blob/main/.github/workflows/workflow.yml) file for all the environment variables that are mentioned in your `default.yml` and `teams/*.yml` YAML files. For the environment variables that map to secrets, add the required secrets to your GitHub repository secrets.
+
+6. Push your cloned and modified code to your repo.
+
+7. In GitHub, enable the `Apply latest configuration to Fleet` GitHub Actions workflow, and run workflow manually. Now, when anyone pushes a new commit, the action will run and update Fleet. In addition, the workflow will run every 24 hours to ensure that the Fleet configuration is in sync with your git repo.
```

Comment:
```suggestion
7. In GitHub, enable the `Apply latest configuration to Fleet` GitHub Actions workflow, and run workflow manually. Now, when anyone pushes a new commit, the action will run and update Fleet.
```

I see what you're after with this! If any change is made outside of the repo (UI/API/fleetctl), Fleet will get reset (sync w/ the repo).

I think we can cut this feature for v1. 

Why? I think it adds more UX complexity. More for the user to keep track of and more for us to explain.

The user can manually trigger the GitHub action if they want to sync right? If so, I think starting w/o the automated run every 24 hours is simpler. The action runs on merge and when you trigger it.

We can get fancy later.

Reply:
Done